### ISSUE TITLE
Something about the label

### ISSUE THREAD

**Issue by jsjs0827** [zhaoweicai/mscnn#66]
I see the MSCNN\mscnn-master\examples\kitti_car\mscnn-7s-384-2x trainval_1st.protxotxt,the label
top: "label_1_5x5"
  top: "label_1_7x7"
  top: "label_2_5x5"
  top: "label_2_7x7"
  top: "label_3_5x5"
  top: "label_3_7x7"
  top: "label_4_5x5"
so many label for just one picture?

**Comment by zhaoweicai** [comment]
Yes, because there are multiple output branches. Each branch needs a label blob.

**Comment by jsjs0827** [comment]
Can you give me the details that how the labels make?I want to train on my own data.Could you give me the way to contact with you.Here is my mailbox:475975144@qq.com
